modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -211,4 +211,52 @@
       end
     end
   end
+
+  path '/veterans/{veteranId}/526/{id}/getPDF' do
+    get 'Returns filled out 526EZ form as PDF' do
+      tags 'Disability'
+      operationId 'get526Pdf'
+      security [
+        { productionOauth: ['system/claim.read', 'system/claim.write'] },
+        { sandboxOauth: ['system/claim.read', 'system/claim.write'] },
+        { bearer_token: [] }
+      ]
+      consumes 'multipart/form-data'
+      produces 'application/json'
+
+      parameter name: 'veteranId',
+                in: :path,
+                required: true,
+                type: :string,
+                example: '1012667145V762142',
+                description: 'ID of Veteran'
+
+      let(:veteranId) { '1013062086V794840' } # rubocop:disable RSpec/VariableName
+      let(:Authorization) { 'Bearer token' }
+      pdf_description = <<~VERBIAGE
+        Returns a filled out 526EZ form for a disability compensation claim (21-526EZ).
+
+        This endpoint can be used to generate the PDF based on the request data in the case that the submission was not able to be successfully auto-established. The PDF can then be uploaded via the Benefits Intake API to digitally submit directly to the Veterans Benefits Administration's (VBA) claims intake process.
+      VERBIAGE
```

Comment:
Is `Benefits Intake API` suppose to be a link to that page? looks like it links to https://staging-developer.va.gov/explore/benefits/docs/benefits?version=current